### DISCUSSION TITLE
建议增加音量锁定功能

### DISCUSSION THREAD

**Discussion by lxl217114** [the1812/Bilibili-Evolved#4085] [功能建议 / Ideas]
目前bilibili每次进入视频播放，音量是100%。
![b](https://user-images.githubusercontent.com/54263659/227125813-d58a3c19-36fa-4523-b5f0-396f794b5592.jpg)

在系统音量不变的前提下，大部分应用音量是合适的。
而bilibili的网页默认100%音量，就会声音太大了，每次要手动去调整播放器的音量大小。

**Comment by Geeyun-JY3** [comment]
实测登录后是会记忆上一个视频的音量的。

另外应该算重复：#3165
